e3sm_diags/plot/annual_cycle_zonal_mean_plot.py

```diff
@@ -109,7 +109,8 @@ def _add_colormap(
     # Add the contour plot
     # --------------------------------------------------------------------------
     ax = fig.add_axes(DEFAULT_PANEL_CFG[subplot_num], projection=None)
-    var = var.transpose("lat", "time")
+    # var = var.transpose("lat", "time")
+    var = var.transpose(var.dims[1], var.dims[0])
```

Comment:
One SST data set has "latitude" instead of "lat" as dimension name. The code change voided using the dimension name explicitly.